integration_tests/src/test/kotlin/integration/MultipleApksIT.kt

```diff
@@ -0,0 +1,29 @@
+package integration
+
+import FlankCommand
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import run
+
+class MultipleApksIT {
+    private val name = this::class.java.simpleName
```

Comment:
This is a localvariable but in cases below it resides within the function call. lets be consistent. One or the other .

Reply:
good point, thanks!